.github/workflows/build.yml

```diff
@@ -65,6 +65,11 @@ jobs:
       with:
         command: build
 
+    - name: "Run cargo doc"
+      uses: actions-rs/cargo@v1
+      with:
+        command: doc
+
```

Comment:
I think we should do 3 runs of `cargo doc` here:

1. `cargo doc`
    - default flags, on nightly
2. `RUSTDOCFLAGS="--cfg docsrs" cargo doc`
    - flags docs.rs will use, on nightly
3. `cargo +stable doc --no-default-features --features=external_asm,instructions`
    - make sure we can build the docs with the stable compiler

Reply:
I decided to just have (1) and (3), now that we don't have the `docsrs` stuff.